addons/hr_payroll/models/hr_payslip.py

```diff
@@ -0,0 +1,581 @@
+#-*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, timedelta
+from dateutil import relativedelta
+import time
+
+from odoo import api, fields, models, tools
+from odoo.tools.translate import _
```

Comment:
now we can directly import translation from odoo . like
from odoo import api, fields, models, tools, _